README.md

```diff
@@ -426,17 +428,27 @@ You can also choose the `services.cloud.sap.com/forceRotate` annotation (value d
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
-## Multitenancy
-You can configure the SAP BTP service operator to work with more than one subaccount in the same Kubernetes cluster. This means that different namespaces can be connected to different subaccounts.
-The association between a namespace and a subaccount is based on a different set of credentials configured for different namespaces.
+## Working with Multiple Subaccounts
+
+By default, a Kubernetes cluster is related to one subaccount (see step 4 of the [Setup](#setup) section.
+The SAP BTP service operator can also be configured to manage multiple subaccounts within a single Kubernetes cluster. In this more complex use case, each resource is linked to a specific subaccount using a unique set of credentials. To connect a resource to a subaccount:
```

Comment:
here we ignore the second level - associating an entire namespace to a subaccount.
e.g.,
"however, it is also possible to configure the SAP BTP service operator to work with more than one subaccount, either to connect an different namespaces to work with different subaccounts, and in more complex use cases, to create within the same namespace service instances in different subaccounts."
The configuration of working with multiple subaccounts is based on using a unique set of credentials, to connect with each subaccount:"